### PR TITLE
conversation assist bot template

### PR DESCRIPTION
**Pull request by cshape**

### CODE REVIEW

File: bot-samples/conv-assist-template/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Instructions
+
+1. [Import the ReplacementCardHelper.json file](https://developers.liveperson.com/conversation-builder-bots-bot-basics.html#import-a-bot) into your account.
+2. Verify that `ask about replacement card` is the assigned intent in the `replace card starter` interaction. If it isn't, then manually assign it.
+3. Select your bot agent as the agent connector for the bot.
+4. Go into the Conversation Assist settings and turn on the `ReplacementCardHelper` bot as a recommendation source for the skill associated with your test campaign.
+5. Use the [Web Messaging emulator](https://developers.liveperson.com/web-messaging/emulator.html) to conduct a test chat with yourself. Have the consumer ask something like "I need a new card" which will trigger the Conversation Assist suggestion to the agent.
+6. As the agent, click the suggestion to delegate the conversation to the bot.
+7. As the consumer, go through the conversational flow.
+8. Once the conversation is completed, go into Bot Analytics -> Custom Events to see the logging for the conversation.
```
Comment (JasonSowers) [inline bot-samples/conv-assist-template/README.md:25]: screenshot may help here, up to you though

File: bot-samples/conv-assist-template/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Instructions
+
+1. [Import the ReplacementCardHelper.json file](https://developers.liveperson.com/conversation-builder-bots-bot-basics.html#import-a-bot) into your account.
+2. Verify that `ask about replacement card` is the assigned intent in the `replace card starter` interaction. If it isn't, then manually assign it.
+3. Select your bot agent as the agent connector for the bot.
+4. Go into the Conversation Assist settings and turn on the `ReplacementCardHelper` bot as a recommendation source for the skill associated with your test campaign.
+5. Use the [Web Messaging emulator](https://developers.liveperson.com/web-messaging/emulator.html) to conduct a test chat with yourself. Have the consumer ask something like "I need a new card" which will trigger the Conversation Assist suggestion to the agent.
+6. As the agent, click the suggestion to delegate the conversation to the bot.
+7. As the consumer, go through the conversational flow.
```
Comment (JasonSowers) [inline bot-samples/conv-assist-template/README.md:24]: Both of these seems vague, especially the agent piece.  Might need some elaboration and possible a screenshot

Comment (cshape) [reply bot-samples/conv-assist-template/README.md]: updated to add screenshot.

File: bot-samples/conv-assist-template/README.md (hunk 1 above)
Comment (cshape) [reply bot-samples/conv-assist-template/README.md:26]: I think these instructions should be fine. Especially with the added screenshot for #7 and the included video in the README.

**Review by JasonSowers — APPROVED** [review]

**Pull request merged**